test/unit_test/runtime/distribution_test.cpp

```diff
@@ -0,0 +1,587 @@
+#include <numeric>
+#include <units/random.h>
+#include <units/physical/si/length.h>
+#include <catch2/catch.hpp>
+
+using namespace units;
+using namespace units::physical::si;
+
+TEST_CASE("uniform_int_distribution")
+{
+  using Rep = std::int64_t;
```

Comment:
Please do not name types from the big letter

Reply:
Since those are reserved for concepts, correct?